react/features/filmstrip/components/native/TileView.js

```diff
@@ -103,7 +103,15 @@ class TileView extends Component<Props> {
     render() {
         const { _height, _width, onClick } = this.props;
         const rowElements = this._groupIntoRows(this._renderThumbnails(), COLUMN_COUNT);
-        const pageOrderedThumbnails = this._groupThumbnailsByPages(rowElements);
+        const mainUser = this.props._participants[1];
```

Comment:
Please use `inFocusUser` and `localUser` naming here, too.
Also add a TODO comment to make inFocusUser settable by tabletparty later.
We need to put always to local participant to the self frame, is she always the first in the participant list?

Reply:
Local participant is always the first one, but when we implement the shorting ticket, we will modify the selection logic.